dynomitemanager/src/main/java/com/netflix/dynomitemanager/defaultimpl/DynomitemanagerConfiguration.java

```diff
@@ -704,11 +713,23 @@ public String getCommaSeparatedCassandraHostNames() {
     public boolean isEurekaHostSupplierEnabled() {
 	return configSource.get(CONFIG_IS_EUREKA_HOST_SUPPLIER_ENABLED, DEFAULT_IS_EUREKA_HOST_SUPPLIER_ENABLED);
     }
-    
+
     // Redis compatibility
     @Override
     public String getRedisCompatibleEngine() {
 	return configSource.get(CONFIG_DYNO_REDIS_COMPATIBLE_ENGINE, DEFAULT_REDIS_COMPATIBLE_ENGINE);
     }
-    
+
+    // Redis
+    // =====
+
+    /**
+    * Get the full path to the redis.conf configuration file.
+    * @return the {@link String} full path to the redis.conf configuration file
+    */
+    @Override
+    public String getRedisConf() {
```

Comment:
An alternative approach is to import `IConfigSource` into `RedisStorageProxy`. 

Then I can move `getRedisConf()` into `RedisStorageProxy` as well.

Does this sound like a reasonable approach?

The above is required because I need to allow a user to configure the path to `redis.conf`. The ability to get configuration properties is provided by `IConfigSource` (which is implemented by `DefaultConfigSource`). `DefaultConfigSource` gets properties from one of:

- `SimpleDBConfigSource`: SimpleDB
- `PropertiesConfigSource`: the `dynomitemanager.properties` file
- `SystemPropertiesConfigSource`: via CLI `-Dproperty=value`